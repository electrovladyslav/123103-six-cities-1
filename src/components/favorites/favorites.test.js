import React from "react";
import renderer from "react-test-renderer";

import Favorites from "./favorites.jsx";

it(`Favorites component renders correctly`, () => {
  const tree = renderer
    .create(
        <Favorites
          loadFavorites={jest.fn().mockImplementation(() => Promise.resolve())}
          favorites={[]}
          favoritesOrderedByCity={{}}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
