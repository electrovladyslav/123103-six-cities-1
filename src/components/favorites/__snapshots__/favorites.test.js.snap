// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites component renders correctly 1`] = `"Please wait for data loading."`;
